cla_backend/settings/base.py

```diff
@@ -270,6 +271,7 @@ def env_var_truthy_intention(name):
     },
     "loggers": {
         "": {"handlers": ["console"], "level": "DEBUG", "propagate": True},
+        "console.direct": {"handlers": ["console"], "propagate": True},
```

Comment:
My knowledge of the python/django logging is pretty weak however when I did 
```
logger = logging.getLogger("")
logger.debug
```
Nothing got printed to the std out, I thought because of the `"level": "DEBUG"` that this logging handler will be used only when in debug mode but I think i might have been wrong.

Reply:
What about `logger = logging.getLogger(__name__)`?
That's what we generally use, which I think would be caught by the root handler.

For the levels, I think that's a minimum level, and `DEBUG` is the lowest so a handler set for that should pick up everything above, including `INFO`, I think